select_multiple_field/forms.py

```diff
@@ -57,7 +57,12 @@ def to_python(self, value):
         Method also handles lists and strings
         """
         if (value == self.empty_value) or (value in self.empty_values):
-            return self.empty_value
+            try:
+                # must be iterable
+                it = iter(self.empty_value)
```

Comment:
Why would `empty_value` not be an iterable?

Wouldn't it be better instead to make sure `empty_value` is iterable when setting it, in the constructor?